aten/src/ATen/core/Error.cpp

```diff
@@ -15,18 +15,18 @@ Error::Error(SourceLocation source_location, std::string err)
   , what_(str(err, " (", source_location, ")\n", get_backtrace(/*frames_to_skip=*/2)))
   {}
 
+static Warning::handler_t global_warning_handler = &Warning::print_warning;
```

Comment:
Why `static`? Would an anonymous namespace do the trick? That would be more common in C++.

Reply:
It turns out I lost this change when I landed the fbcode change. Which is for the best, because this change wasn't actually necessary (it was me trying to fix Caffe2 Windows which actually was fixed properly later.)